lib/ansible/cli/__init__.py

```diff
@@ -8,6 +8,16 @@
 import locale
 import os
 import sys
+import argparse
+
+import typing
```

Comment:
FYI, the commonly used style across the project is doing
```suggestion
import typing as t
```
and then, accessing whatever's needed through that. Like `t.Callable`, for example.

Reply:
The question here is if we want to use `import typing as t` instead of `from typing import Callable` so we're able to use `Callable` directly like `list`, `str` and so on.

Reply:
`Callable` should be imported from `collections.abc`, not `typing`.